### PR TITLE
add name to dive shop table

### PR DESCRIPTION
**Pull request by Nnanyielugo**

### CODE REVIEW

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+revision = '44d4df0fe283'
+down_revision = '2f4db5864a2d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('dive_shop', sa.Column('name', sa.String(), nullable=True))
+    op.alter_column('user', 'unit',
```
Comment (mjmayank) [inline migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:22]: can you delete this line

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+               existing_type=sa.VARCHAR(),
+               nullable=False)
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('user', 'unit',
```
Comment (mjmayank) [inline migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:30]: this one also

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 1 above)
Comment (Nnanyielugo) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:22]: the entire upgrade function?

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 2 above)
Comment (Nnanyielugo) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:30]: This entire function as well?

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 1 above)
Comment (mjmayank) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:22]: the entire thing with unit

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 2 above)
Comment (mjmayank) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:30]: same - entire thing with unit

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 1 above)
Comment (Nnanyielugo) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py:22]: Got it.

File: migrations/versions/44d4df0fe283_add_name_to_dive_shop.py (hunk 2 above)
Comment (Nnanyielugo) [reply migrations/versions/44d4df0fe283_add_name_to_dive_shop.py]: Done

**Review by mjmayank — APPROVED** [review]

**Pull request merged**